### DISCUSSION TITLE
Admin Notifications Discord Role

### DISCUSSION THREAD

**Discussion by Kilometres** [TAServers/issues#19] [Ideas]
# Admin Notifications Discord Role
While `playerbase` is still being developed (or just an idea), we could add a role so that when members need an admin but there are none in the server they are able to earlier ping them and get someone's attention.

**Comment by ELPatitoLoco** [comment]
I mean unless its for discord help admins don't have rcon so if someon is lagging better ping a system admin, but still cool idea.

**Reply by Kilometres** [reply]
I meant so they could get on the server. And new relay will have ulx commands for other than sys admin.

**Comment by Derpius** [comment]
Forgot about this one, I'll create a gmod-admin pingable role now

Might also remove the capability roles as they can't be consistently applied, but will take a vote for that
